budgeteer-admin-interface/src/main/java/de/adesso/budgeteer/web/pages/administration/BudgeteerAdministrationOverview.java

```diff
@@ -0,0 +1,317 @@
+package de.adesso.budgeteer.web.pages.administration;
+
+import de.adesso.budgeteer.web.pages.base.basepage.BasePage;
+import de.adesso.budgeteer.web.pages.base.basepage.breadcrumbs.BreadcrumbsModel;
+import de.adesso.budgeteer.web.pages.user.login.LoginPage;
+import org.apache.wicket.Component;
+import org.apache.wicket.ajax.AjaxEventBehavior;
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.markup.html.WebMarkupContainer;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.form.*;
+import org.apache.wicket.markup.html.link.Link;
+import org.apache.wicket.markup.html.list.ListItem;
+import org.apache.wicket.markup.html.list.ListView;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.apache.wicket.model.StringResourceModel;
+import org.apache.wicket.model.util.ListModel;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.wickedsource.budgeteer.persistence.user.UserEntity;
+import org.wickedsource.budgeteer.service.project.ProjectBaseData;
+import org.wickedsource.budgeteer.service.project.ProjectService;
+import org.wickedsource.budgeteer.service.user.MailAlreadyInUseException;
+import org.wickedsource.budgeteer.service.user.User;
+import org.wickedsource.budgeteer.service.user.UserService;
+import org.wickedsource.budgeteer.service.user.UsernameAlreadyInUseException;
+import org.wickedsource.budgeteer.web.BudgeteerSession;
+import org.wickedsource.budgeteer.web.ClassAwareWrappingModel;
+import org.wickedsource.budgeteer.web.Mount;
+import org.wickedsource.budgeteer.web.components.customFeedback.CustomFeedbackPanel;
+import org.wickedsource.budgeteer.web.components.user.UserRole;
+import org.wickedsource.budgeteer.web.pages.base.delete.DeleteDialog;
+import org.wickedsource.budgeteer.web.settings.BudgeteerSettings;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import static org.wicketstuff.lazymodel.LazyModel.from;
+import static org.wicketstuff.lazymodel.LazyModel.model;
+
+@Mount("/administartion-menu")
+public class BudgeteerAdministrationOverview extends BasePage {
+
+    @SpringBean
+    private UserService userService;
+
+    @SpringBean
+    private ProjectService projectService;
+
+    @SpringBean
+    private BudgeteerSettings settings;
+
+    private User thisUser = BudgeteerSession.get().getLoggedInUser();
+
+    private CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedback");
+
+    public BudgeteerAdministrationOverview() {
+        add(feedbackPanel);
+        add(createUserList("userList", new ListModel<>(userService.getAllUsers())));
+        add(createProjectList("projectList", new ListModel<>(projectService.getAllProjects())));
+    }
+
+
+    private ListView<User> createUserList(String id, IModel<List<User>> model) {
+
+        return new ListView<User>(id, model) {
```

Comment:
Maye it would make sense, to create an own component.

Reply:
I see what you mean, but I think it's better to keep the ListView here. A component wouldn't be reusable in any other place and the method isn't all that long in my opinion.